models/issues/issue_list.go

```diff
@@ -255,16 +259,14 @@ func (issues IssueList) LoadProjects(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		for _, project := range projects {
-			projectMaps[project.IssueID] = project.Project
-		}
 		left -= limit
 		issueIDs = issueIDs[limit:]
+		for _, projectIssue := range projects {
+			issue := issueMap[projectIssue.IssueID]
+			issue.Projects = append(issue.Projects, projectIssue.Project)
```

Comment:
Maybe we need to check whether `projectIssue.Project` is already in issue.Projects here.